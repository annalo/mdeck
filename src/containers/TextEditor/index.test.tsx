import React from "react";
import { render } from "@testing-library/react";
import { HelmetProvider } from "react-helmet-async";

import { TextEditor } from ".";

const renderComponent = () =>
  render(
    <HelmetProvider>
      <TextEditor markdown="markdown text" setMarkdown={() => {}} />
    </HelmetProvider>
  );

describe("<TextEditor />", () => {
  it("should match the snapshot", () => {
    const component = renderComponent();
    expect(component.container.firstChild).toMatchSnapshot();
  });
});
