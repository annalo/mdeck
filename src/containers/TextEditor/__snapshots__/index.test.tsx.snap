// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextEditor /> should match the snapshot 1`] = `
.c0 {
  max-height: 100%;
}

<textarea
  class="c0"
>
  markdown text
</textarea>
`;
